prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/csp.js

```diff
@@ -40,9 +48,14 @@ export default contentSecurityPolicyBuilder({
     connectSrc: [
       "'self'",
       '*.api.frank',
-      `${ip.address()}:${process.env.HTTP_ONE_APP_DEV_CDN_PORT || 3001}`,
-      `localhost:${process.env.HTTP_ONE_APP_DEV_CDN_PORT || 3001}`,
+      // used for local development
+      `${ip.address()}:3001`,
```

Comment:
If this variable is undefined (which it is in any production set up) then the CSP is invalid.

I think with the normalize updates you mentioned this will now work as intended. The issue was that the string `undefined` was used as the value to `HTTP_ONE_APP_DEV_CDN_PORT` and hence the logic for default port 3001 was never reached.